### DISCUSSION TITLE
Record to local disk and flush to network storage periodically?

### DISCUSSION THREAD

**Discussion by nanderson97651** [blakeblackshear/frigate#9302] [Ask A Question]
My current configuration has Frigate running on a RPi4 with Coral and small SSD with storage being a mapped NFS share living on my NAS.  I've been giving more thought to the power consumption of devices in my house and am trying to reduce the amount of time the hard drives in the NAS are running, a challenge with Frigate writing so often.

I've already taken steps to reduce the amount of recording happening by tweaking my motion detection in config.yaml but I was wondering if there's a way-in Frigate or Linux in general-to have the recording take place on the local disk and then periodically shuttle that over to the NAS.  The biggest challenge I expect is to have Frigate be able to recognize the two locations (or make it entirely transparent to Frigate).  I wouldn't want to lose the ability to look at past days clips in the UI if possible.

**Comment by NickM-27** [comment]
this is possible using something like fuse.fs, Unraid does this automatically saving to an SSD cache pool and then writing recordings to HDD array once that cache pool is full

**Reply by nanderson97651** [reply]
I run a Synology DS920+ which does have option for SSD cache.  I looked into it and it seems like it doesn't give much benefit for large uploads/downloads or sequential I/O.  I can always try to grab a cheap NVME drive and give it a shot but if there's a no-cost way to do it with the existing hardware I would be interested.

Can you point me in the right direction for some documentation of the relevant fuse.fs features?
